lifetiles-graph/src/main/java/nl/tudelft/lifetiles/graph/model/GraphContainer.java

```diff
@@ -137,8 +136,12 @@ public final void setVisible(final Set<Sequence> visibleSequences) {
     public final Set<SequenceSegment> getVisibleSegments(final int position) {
 
         Set<SequenceSegment> copy = new TreeSet<SequenceSegment>();
-        for (SequenceSegment seg : segmentBuckets.getSegments(position)) {
-            copy.add(seg.clone());
+        try {
```

Comment:
The `try ... catch`-block should be inside the loop so that it can be debugged more easily.